.travis.yml

```diff
@@ -59,6 +72,12 @@ jobs:
       env: TOXENV=py36 PYTHON='3.6' PYENV_VERSION='system'
       after_success:
         - bash <(curl -s https://codecov.io/bash) -e TOX_ENV
+    - stage: test
```

Comment:
Since we are adding 3.7 does it make sense to remove 3.4?

Reply:
Nope, py34 is still in production for till April 2019.